src/server/api/redirect.ts

```diff
@@ -36,7 +36,7 @@ async function getLongUrlFromStore(shortUrl: string): Promise<string> {
     // Cache failed, look in database
     const longUrl = await getLongUrlFromDatabase(shortUrl)
     try {
-      cacheShortUrl(shortUrl, longUrl)
+      await cacheShortUrl(shortUrl, longUrl)
```

Comment:
we actually want to redirect ASAP, and not await on this operation.

Reply:
how about `cacheShortUrl(shortUrl, longUrl).catch(...)` instead?

good catch.

Reply:
Changed to the async catch